src/pyodide/BUILD.bazel

```diff
@@ -0,0 +1,50 @@
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+load(":patch_pyodide_js.bzl", "patch_pyodide_js")
+load("@workerd//:build/wd_ts_bundle.bzl", "wd_ts_bundle")
+
+
+copy_file(
```

Comment:
need to format the file.

Reply:
What do we use for starlark code formatting? I ran black on this.